modules/javafx.controls/src/test/java/test/javafx/scene/chart/AreaChartTest.java

```diff
@@ -534,8 +534,8 @@ private List<Point2D> findDataPointsFromPathLine(AreaChart<Number, Number> areaC
                 .filter(pathElement -> pathElement instanceof LineTo)
                 .map(pathElement -> (LineTo) pathElement)
                 .map(lineTo -> new Point2D(
-                        xAxis.getValueForDisplay(lineTo.getX()).doubleValue(),
-                        yAxis.getValueForDisplay(lineTo.getY()).doubleValue())
+                        Math.ceil(xAxis.getValueForDisplay(lineTo.getX()).doubleValue()),
+                        Math.ceil(yAxis.getValueForDisplay(lineTo.getY()).doubleValue()))
```

Comment:
would it make more sense to either do a Math.round(), or better yet - write a utility that computes array equality of Point2D's with some non-zero tolerance?

Do we have more tests like this that might warrant a new utility?

Reply:
AFAIK, we do not have other tests with that problem.
I tried to keep the diff small, but nothing against writing a better method to compare the points here with a delta.

Reply:
I am not sure why this is needed here, so my concern stems from the fact that `ceil()` might introduce a large bias (0.000001 -> 1.0), and `rnd()` _seems_ to be a better choice.

But then again, ceil() or rnd() might be a wrong thing to do if we are dealing with scaled coordinates and we don't expect the whole integers, but instead are dealing with small differences between expected and obtained values (in which case a better comparison is required).

Reply:
I can also use `round` here. Seems to be a bit safer, I agree.

Reply:
thank you.

one more question - is it possible to set fractional scale with the StubToolkit, and how would that work in the StubFontLoader?  Does it mean we'll be rounding the scaled values?

Reply:
You mean for the font size? That will as far as I can see just work, although not 100% sure what the rendering, e.g. a Label will do with it